liiatools/datasets/social_work_workforce/sample_data.py

```diff
@@ -0,0 +1,213 @@
+import random
+import string
+from datetime import date, datetime, timedelta
+import pytz
+
+from sfdata_stream_parser.events import StartElement as S, EndElement as E, TextNode as T
+
+from liiatools.datasets.social_work_workforce.schema import Schema
+
+
+def TE(tag: str, text):
```

Comment:
Similar to my comment about the imports, defining a function like this isn't self-documenting. What does TE stand for? I assume TextElement, but naming it like this creates inconsistencies in your standard naming pattern. Most functions you name_like_this, but here you're abbreviating and using capitals. Any reason for the difference?

Reply:
Just as above, using the same process that Kaj did but happy to change for readability